pom.xml

```diff
@@ -347,7 +358,7 @@ The MIT License (MIT)
           <plugin>
             <groupId>org.codehaus.mojo</groupId>
             <artifactId>sonar-maven-plugin</artifactId>
-            <version>3.4.0.905</version>
+            <version>3.6.0.1398</version>
```

Comment:
@scrobot if this is not related to the issue you are solving, please revert this change (if it is, please explain)